pyproject.toml

```diff
@@ -13,8 +13,7 @@ packages = [{include = "autoblocks"}]
 
 [tool.poetry.dependencies]
 python = "^3.8.1"
-httpx = "^0.24.0"
-wrapt = "^1.15.0"
+httpx = ">=0.24.0"
```

Comment:
relaxing this so people can install e.g. httpx==0.25.0. can't update this ourselves until https://github.com/Colin-b/pytest_httpx/issues/122 is fixed tho